testsuite/python/collision_detection.py

```diff
@@ -79,7 +80,7 @@ def test_bind_centers(self):
         self.s.part.add(pos=(0, 0, 0), id=0)
         self.s.part.add(pos=(0.1, 0, 0), id=1)
         self.s.part.add(pos=(0.1, 0.3, 0), id=2)
-        self.s.integrator.run(0)
+        self.s.integrator.run(1)
```

Comment:
Wouldn't it be cleaner to add an interface function to explicitly execute the collision detection ?
Like
```c++
void collide() {
 on_observable_calc();
 short_range_loop(...);
 handle_collisions();
}
```
should work right?